static/js/popovers.js

```diff
@@ -449,6 +511,20 @@ exports.register_click_handlers = function () {
         show_user_info_popover(this, user, message);
     });
 
+    $("#main_div").on("click", ".user-group-mention", function (e) {
+        var id = $(this).attr('data-user-group-id');
+        if (id === '*') {
+            return;
+        }
```

Comment:
Why do you have a check for `*` here?  

(For context, `*` is a special value used in the older "user mention" feature for "wildcard mentions" -- i.e. mentioning everyone in the stream).